### DISCUSSION TITLE
一个很奇怪的事情

### DISCUSSION THREAD

**Discussion by MoYuan-CN** [Meloong-Git/PCL#1476] [提问] [answered]
咋 PCL2 出现在 [这里](https://www.onlinedown.net/soft/10042217.htm) 了

**Comment by Chiloven945** [comment]
6，什么离谱的东西（

**Comment by LTCatt** [comment, marked as answer]
这些软件网站的“自行搬运”无法阻止，也不用管，算了（

**Comment by iPodtouch6557** [comment]
> 咋 PCL2 出现在 [这里](https://www.onlinedown.net/soft/10042217.htm) 了

哈哈哈，我最近发现的bug实验文件就是这个下的
